modules/quick-starts-highlighting-reference.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-highlighting-reference_{context}"]
+= Quick start highlighting markdown reference
+
+The highlighting, or hint, feature enables Quick Starts to contain a link that can highlight and animate a component of the web console.
+
+The markdown syntax contains:
+
+* bracketed link text
+* the `highlight` keyword, followed by
+* the ID of the element that you want to animate
```

Comment:
I've generally seen bulleted lists capitalize the first letter of each item. ISG check seems to agree though I can't find a rule about it from a quick search. 🤷